github/admin.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "fmt"
+
+// AdminService handles communication with the admin related methods of the
+// GitHub API. These API routes are normally only accessible for GitHub
+// Enterprise installations.
+//
+// GitHub API docs: https://developer.github.com/v3/enterprise/
+type AdminService service
+
+// TeamLDAPMapping represents the mapping between a GitHub team and an LDAP
+// group.
```

Comment:
Minor thought. If it were me, I wouldn't create a second line just for one word. [Why not](https://twitter.com/rob_pike/status/563801489190043648) keep it together with the rest of the sentence?

Either way is completely fine.